features/step_definitions/menu_steps.rb

```diff
@@ -1,21 +1,23 @@
+# frozen_string_literal: true
+
 Then /^I should see menu items$/ do |items|
-  items.raw.each do |item|
-    within '#mainmenu' do
-      assert has_css? 'li', :text => item[0]
-    end
+  within '#mainmenu' do
+    assert_equal items.raw.flatten, find_all('#mainmenu .pf-c-nav__list > .pf-c-nav__item > .pf-c-nav__link').map(&:text)
   end
 end
 
-Then /^I should not see menu items$/ do |items|
```

Comment:
This step not needed anymore?

Reply:
I had to fix the other step `I should see menu items` and now it will fail unless _all_ menu items match, so it already assert missing menu items.

Reply:
maybe call the other step `I should see only menu items`

Reply:
That's `I should see menu items` 😄